### DISCUSSION TITLE
Adonis as backend and api at the same time

### DISCUSSION THREAD

**Discussion by Malin88** [adonisjs/core#2892] [General]
Hi,
I have a quick question. If I would like to create a public frontend application in next.js but the admin panel should be in the normal adonis template then if I choose the web structure, can I still call api endpoints from next.js? So some endpoints should run as API and in a standard way at the same time.

**Comment by thetutlage** [comment]
Yes you can. It's just that the API project structure doesn't install some of the packages like `@adonisjs/session`, `@adonisjs/view`. However, you will need these packages for your admin panel.

**Reply by Malin88** [reply]
Great! Thanks! :)
